build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -30,8 +30,7 @@
     tf_models.MINILM_L12_H384_UNCASED_INT32_SEQLEN128,
     tf_models.RESNET50_TF_FP32,
     torch_models.MODEL_CLIP_TEXT_SEQLEN64_FP32_TORCH,
-    # Disabled due to https://github.com/openxla/iree/issues/11447.
-    #torch_models.MODEL_UNET_2D_FP32_TORCH,
+    torch_models.MODEL_UNET_2D_FP32_TORCH,
```

Comment:
The x86 benchmarks now take 46 mins to run. Seems fine to me but we probably need to fix this before adding more models (either using more machines to run benchmarks or trying to cut some experimental benchmarks)

Reply:
Ok I'll hold off on adding more x86 benchmarks.